settings.gradle.kts

```diff
@@ -1,3 +1,5 @@
 include("android")
 include("coroutines")
 include("stdlib")
+
+rootProject.name = "Extensions"
```

Comment:
Can you lower case this? It might cause the artifact to be capitalized and that causes weird issues